src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/Handlers/AbstractEvaluationCommandLineHandler.cs

```diff
@@ -156,6 +156,18 @@ public void ApplyProjectBuild(IComparable version, IProjectChangeDiff difference
 
         protected abstract void UpdateInContext(string fullPath, IImmutableDictionary<string, string> previousMetadata, IImmutableDictionary<string, string> currentMetadata, bool isActiveContext, IProjectDiagnosticOutputService logger);
 
+        private bool ItemInCurrentConfiguration(string includePath, IImmutableDictionary<string, IImmutableDictionary<string, string>> metadata)
+        {
+            if (metadata.TryGetValue(includePath, out IImmutableDictionary<string, string> itemMetadata)
+                && itemMetadata.TryGetValue("ExcludeFromCurrentConfiguration", out string excludeFromCurrentConfiguration)
+                && StringComparers.PropertyValues.Equals(excludeFromCurrentConfiguration, "true"))
```

Comment:
I think we can use `CompileSchema.ExcludeFromCurrentConfigurationProperty` here to avoid the literal string here. There's also a `GetBoolProperty` helper method, from memory.

Something like:

```suggestion
            if (metadata.TryGetValue(includePath, out IImmutableDictionary<string, string> itemMetadata)
                && itemMetadata.GetBoolProperty(CompileSchema.ExcludeFromCurrentConfigurationProperty) is true)
```